public/app/features/logs/components/LogDetailsRow.tsx

```diff
@@ -21,6 +22,7 @@ export interface Props extends Themeable2 {
   showDetectedFields?: string[];
   onClickShowDetectedField?: (key: string) => void;
   onClickHideDetectedField?: (key: string) => void;
+  row?: LogRowModel;
```

Comment:
Is there a reason that this is optional? Because it seems that we are passing row everywhere where `LogDetailsRow` component is used and in `LogDetails`, `row` is required. Do we could make it mandatory.